react-live.css

```diff
@@ -92,3 +92,9 @@
 .token.deleted {
   color: red;
 }
+
+.tabGuarded{
```

Comment:
Looked into it further and deleting the `overflow: hidden` didn't work for me.  It's the change in implementation of previously having a wrapper but now dealing directly with the `Editor`.  Could be wrong though, but it's what I am seeing.